index.html

```diff
@@ -1196,39 +1197,40 @@ <h2><dfn>RTCQuicStreamState</dfn> Enum</h2>
               <td><dfn><code id=
               "idl-def-RTCQuicStreamState.closing">closing</code></dfn></td>
               <td>
-                <p>The procedure to close down the QUIC stream has started.
+                <p>The procedure to close down the <code><a>RTCQuicStream</a></code>
+                <var>stream</var> has started.
                 This can happen in multiple ways:</p>
                   <ol>
-                     <li>A STREAM frame with the FIN flag set has not yet been
-                     received and the <code>reset</code> method has been called.
-                     This causes a RST_STREAM frame to be queued for transmission,
-                     the read and write buffers to be cleared and the
-                     <a>[[\Readable]]</a> and <a>[[\Writeable]]</a> slots are set
+                     <li><var>stream</var> has not <a>finished reading</a> and the
+                     <code>reset</code> method has been called. This causes a
+                     RST_STREAM frame to be queued for transmission, the read
+                     and write buffers to be cleared and <var>stream</var>'s
+                     <a>[[\Readable]]</a> and <a>[[\Writeable]]</a> slots to be set
                      to <code>false</code>.</li>
-                     <li>A STREAM frame with the FIN flag set has not yet been
-                     received and the <code>finish</code> method has been called.
+                     <li><var>stream</var> has not <a>finished reading</a> and the
+                     <code>finish</code> method has been called.
                      This causes a STREAM frame with the FIN flag set to be queued
-                     for transmission, the write buffer to be cleared and the
-                     <a>[[\Writeable]]</a> slot is set to <code>false</code>.</li>
-                     <li> The QUIC stream has received STREAM frame with the FIN flag set.</li>
+                     for transmission, the write buffer to be cleared and <var>stream</var>'s
+                     <a>[[\Writeable]]</a> slot to be set to <code>false</code>.</li>
+                     <li><var>stream</var> has <a>finished reading</a>.</li>

```

Comment:
Should this set [[Readable]] to false?

Reply:
Calling finish() causes a transition to "closing" with the Writeable slot false, but the Readable slot remains true until a FIN or RST_STREAM is received.  This is shown in Figure 1.

Reply:
Right. This is saying we transition to closing when we've "finished reading," meaning if the stream has "_read all available data up to the STREAM frame with the FIN bit set_." 

I'm asking if at that point the [[Readable]] slot should transition to false (because there's nothing to read anymore). If we also need to wait for finish() to be called, then it can stay as is.

Reply:
Not sure the current text explains things well. Doing so requires describing the transitions via a combination of current state/event/new state.  WebRTC 1.0 does a reasonable job of this in Section 5.6.4, for example (see: https://rawgit.com/w3c/webrtc-pc/master/webrtc.html#rtcicetransportstate )

Reply:
Yes, it should. Added text to make that clear.

Reply:
Looks good 👍